tests/podman-compose.yml

```
@@ -0,0 +1,59 @@
+version: '3'
+services:
+  graph-node:
+    image: graphprotocol/graph-node
+    userns_mode: "keep-id"
+    user: root
+    ports:
+      - '8000:8000'
+      - '8001:8001'
+      - '8020:8020'
+      - '8030:8030'
+      - '8040:8040'
+    depends_on:
+      - ipfs
+      - postgres
+    environment:
+      postgres_host: postgres
+      postgres_user: graph-node
+      postgres_pass: let-me-in
+      postgres_db: graph-node
+      ipfs: 'ipfs:5001'
+      ethereum: 'mainnet:http://anvil:8545'
+      GRAPH_LOG: info
+  ipfs:
+    image: ipfs/go-ipfs:v0.10.0
+    userns_mode: "keep-id"
+    ports:
+      - '5001:5001'
+    volumes:
+      - ./data/ipfs:/data/ipfs
```

This change is somewhat specific to the fact that I'm on an SELinux-enabled distro, but it's part of the official docker-compose API, so it doesn't hurt.
```suggestion
      - ./data/ipfs:/data/ipfs:Z
```
More details here: https://docs.docker.com/compose/compose-file/05-services/#short-syntax-5

---

On the other hand it causes issues on my computer if I leave it like that